src/compiler/diagnosticMessages.json

```diff
@@ -4313,5 +4313,13 @@
     "Convert named imports to namespace import": {
         "category": "Message",
         "code": 95057
+    },
+    "Covert React.Component to SFC": {
+        "category": "Message",
+        "code": 95058
+    },
+    "Covert React.SFC to PureComponent": {

```

Comment:
`Convert React Stateless Functional Component to a statefull component`

Reply:
@DanielRosenwasser can you help out with these messages..

Reply:
Is `Pure Component` better than `stateful component`?

Refactor provides an action to `PureComponent` but not `Component`, because `PureComponent` has better performance in default, if developers want to use `state`, they can just remove `Pure`.